plugins/woocommerce/client/admin/client/settings-payments/components/payment-extension-suggestion-list-item/payment-extension-suggestion-list-item.tsx

```diff
@@ -81,12 +91,16 @@ export const PaymentExtensionSuggestionListItem = ( {
 					<div className="woocommerce-list__item-after__actions">
 						<Button
 							variant="primary"
-							onClick={ () =>
+							onClick={ () => {
+								if ( !! extension._incentive ) {
```

Comment:
It's exactly what `hasIncentive` checks. How about replacing this with that method?

Reply:
Updated. Thanks!